hashintel/hash
# H-1751: Display count next to filter options on `/drafts` page

## 🌟 What is the purpose of this PR?




This PR adds a numeric value next to each filter option on the `/drafts` page, which indicates the number of entities which would come into view with the filter option enabled/disabled.

## 🔗 Related links





- H-1751

## 🔍 What does this change?




- adds a `FilterSection` component which makes some of the layout for each filter section more re-usable, and defines the filter sections as an array of objects

## Pre-Merge Checklist 🚀

### 🚢 Has this modified a publishable library?




This PR:


### 📜 Does this require a change to the docs?




The changes in this PR:


### 🕸️ Does this require a change to the Turbo Graph?




The changes in this PR:


## ⚠️ Known issues



## 🐾 Next steps



## 🛡 What tests cover this?



Manual testing.

## ❓ How to test this?



1. Go to the `/drafts` page with drafts of different kinds
2. Ensure the numbers next to the filter options are accurate

## 📹 Demo



https://github.com/hashintel/hash/assets/42802102/ae66e764-0611-44fe-a230-7e8b5f2fc860
